components/builder-graph/src/acyclic_package_graph.rs

```diff
@@ -409,4 +568,185 @@ mod test {
 
         let (..) = graph.extend(&package2, true);
     }
+
+    fn make_package(ident: &str, deps: &[&str]) -> originsrv::OriginPackage {
+        let mut package = originsrv::OriginPackage::new();
+        package.set_ident(originsrv::OriginPackageIdent::from_str(ident).unwrap());
+
+        let mut package_deps = RepeatedField::new();
+        for dep in deps {
+            package_deps.push(originsrv::OriginPackageIdent::from_str(dep).unwrap());
```

Comment:
For your consideration: taking into account our future direction for validation of `__Ident`'s, we may also want to consider removing these `.unwrap()` calls here, utilize the `?` operator and return a `Result<originsrv::OriginPackage>`.